packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -1,12 +1,237 @@
+import { VaAccordion } from './'
+import { VaCollapse } from '../va-collapse'
+import { within, userEvent } from '@storybook/testing-library'
+import { sleep } from '../../utils/sleep'
+
+async function DispatchEventWithDelay (element: HTMLElement, callback: (element: HTMLElement) => void, ms: number) {
+  await callback(element)
+  await sleep(ms)
+}
 
 export default {
   title: 'VaAccordion',
   component: VaAccordion,
+  tags: ['autodocs'],
+}
+
+export const Default = () => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return {
+      modelValue: [false, false, false],
+      collapses: [
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+      ],
+    }
+  },
+  template: `
+    <va-accordion v-model="modelValue">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+Default.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const collapses = await canvas.getAllByRole('button', { name: 'Collapse' }) as HTMLElement[]
+  const [lastCollapse] = collapses.slice(-1)
+  collapses.push(lastCollapse)
+
+  for (const collapse of collapses) {
+    await DispatchEventWithDelay(collapse, userEvent.click, 500)
+  }
+}
+
+export const Stateful = () => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return {
+      collapses: [
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+      ],
+    }
+  },
+  template: `
+    <va-accordion stateful>
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+Stateful.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const collapses = await canvas.getAllByRole('button', { name: 'Collapse' }) as HTMLElement[]
+  const [lastCollapse] = collapses.slice(-1)
+  collapses.push(lastCollapse)
+
+  for (const collapse of collapses) {
+    await DispatchEventWithDelay(collapse, userEvent.click, 500)
+  }
 }
 
-export const Default = defineComponent({
-  components: { VaAccordion },
-  template: '<VaAccordion/>',
+export const Multiple = () => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return {
+      modelValue: [false, true, false],
+      collapses: [
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+      ],
+    }
+  },
+  template: `
+    <va-accordion v-model="modelValue" multiple>
```

Comment:
We can actually shortcut template even more with following syntax:

```html
<va-accordion>
      <va-collapse
        v-for="i in 3"
        :key="i"
        header="Collapse"
      >
        Content
      </va-collapse>
    </va-accordion>
```

That way you don't need data even.